Orange/widgets/data/oweditdomain.py

```diff
@@ -508,11 +546,13 @@ def open_editor(self, index):
 
         var = self.domain_model[index]
 
-        editor_index = 2
+        editor_index = 3
         if var.is_discrete:
             editor_index = 0
-        elif var.is_continuous:
+        elif var.is_time:
```

Comment:
Can you add a comment why time variables need to be handled before numeric variables?

Reply:
TimeVariable is also continuous so we need to test if_time first. Will add comment.